### PR TITLE
Summary component, JSON tree component, and better handling of missing metrics or inaccessible backend

### LINKED ISSUE #36: Add a link to directly render raw metrics as JSON

### LINKED ISSUE #43: Better visual feedback prior to initial HTTP GET
Before a microservice receives a GET request, the `metrics.json` associated with that microservice is missing many of the metrics required to render HTTP metrics. These charts should assume values of 0 when the keys are not present to allow the charts to render and show more useful visual feedback.

image

### LINKED ISSUE #50: Implement Summary Screen
Currently, the summary view is a grid of dummy UI cards. Replace this with a component that provides a single heads-up display of the state of a microservice.

Document the backend features needed to implement UX elements that are not currently.

single-service-instance-view

### LINKED ISSUE #51: Remove Dummy Filler Content
The dashboard has sufficient content to make the filler cards unnecessary. Remove them to draw focus on the functional components of the dashboard.

### LINKED ISSUE #53: Provide User Feedback when backend endpoints are not available.
Currently, the app will fail silently in the console:
image

Due to the nature of the reducers, the error does not impact state. However, it is likely inefficient to fire the reducers on the same interval over and over when an endpoint is down.

Possible Improvements: 
1. Present UIKit Notification when the app is unable to reach an endpoint.
2. After a certain number of consecutive failures, turn off polling.
3. Change some visual indicator on the navbar to indicate that polling has stopped.

### PR DESCRIPTION
**Pull request by spmcbride1201**
Resolves #50.
Resolves #36.
Resolves #43.
Resolves #51.
Resolves #53.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -40,7 +41,9 @@ if (process.env.NODE_ENV === 'development') {
       name === 'AnimationDecorator(ComposedDataDecorator(LineChart))'
       || name === 'AnimationDecorator(ComposedDataDecorator(AreaChart))'
       || name === 'Slider'
-      ) return false;
+      || name === 'JSONNestedNode'
+      || name === 'JSONArrow'
```
Comment (tilleryd) [inline src/index.js]: Perhaps we should reference the GitHub issues in a comment.  That way anyone can easily jump to the issue to see the progress.

File: src/store.js [hunk 2]
```diff
@@ -118,7 +150,7 @@ Hook((action, getState) => {
 export default createStore(
   combineReducers({ metrics, settings, routing: routerReducer }),
   applyMiddleware(
-    reduxPromise,
+    // reduxPromise,
```
Comment (tilleryd) [inline src/store.js]: Is reduxPromise moot since you're using Jumpstate?

Comment (spmcbride1201) [inline src/store.js]: Yep, not needed any more. I removed the commented out dead code here.

File: src/index.js (hunk 1 above)
Comment (spmcbride1201) [inline src/index.js]: Great idea!

File: src/components/HTTPConnectionsAreaChart.js [hunk 3]
```diff
@@ -9,31 +9,35 @@ export default function HTTPConnectionsLineChart({ connectionsArr }) {
...
+              minTickGap={10}
+              padding={{ left: 25 }}
+            />
+            <YAxis allowDecimals={false} />
+            <Tooltip />
+          </AreaChart>
+        </ResponsiveContainer>
+      ) : (
+        <h4>HTTP Connections Data Unavailable</h4>
+        )}
```
Comment (spmcbride1201) [inline src/components/HTTPConnectionsAreaChart.js]: Deferring until next PR per our discussion.

File: src/components/JSONComponent.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+import { connect } from 'react-redux';
+
+class JSONComponent extends Component {
+  static propTypes = {
+    metrics: PropTypes.object
+  };
+
+  render() {
+    const { metrics } = this.props;
+    console.log('metrics: ', metrics);
```
Comment (spmcbride1201) [inline src/components/JSONComponent.js]: Taking out console.log and deferring style refactor per our chat.

**Pull request merged**